src/js/components/LocalImplementationProjects.jsx

```diff
@@ -0,0 +1,31 @@
+import useAirtableCMS from "../hooks/useAirtableCMS";
+import Markdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+
+const LocalImplementationProjects = (props) => {
+  const lip = useAirtableCMS({
+    baseID: "app1YqNgXXkVH04nO",
+    tableName: "Local Implementation Projects",
+    keyField: "name",
+    fieldMapping: {
+      name: "Name",
+      desc: "Description",
+      notes: "Notes"
```

Comment:
The "Notes" field is really meant to be internally-facing notes for the content managers; I think we should avoid fetching it here, since it's not content we'd want to display on the site.